upup/pkg/fi/vfs_castore.go

```diff
@@ -931,6 +809,7 @@ func (c *VFSCAStore) storeCertificate(name string, ki *keysetItem) error {
 		}
 	}
 
+	// TODO remove this
```

Comment:
Is this referring to the code block below? when should (or could) it get removed?

Reply:
Yes. To remove writing of the legacy format I'd have to  fix the unit test that relies on them, verify there is nothing else left reading them, and write code to delete them from upgraded keystores. To allow kops version rollback this should only be done in a subsequent kops release.

Reply:
ok, can we update the comment to reflect that?

```suggestion
	// TODO remove this when we drop support for rolling back to kops versions that support legacy-format keypairs
```